src/main/resources/view/PersonListCard.fxml

```diff
@@ -9,10 +9,12 @@
 <?import javafx.scene.layout.Region?>
 <?import javafx.scene.layout.VBox?>
 
+<?import javafx.scene.control.TextArea?>
 <HBox id="cardPane" fx:id="cardPane" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1">
   <GridPane HBox.hgrow="ALWAYS">
     <columnConstraints>
-      <ColumnConstraints hgrow="SOMETIMES" minWidth="10" prefWidth="150" />
+      <ColumnConstraints hgrow="SOMETIMES" percentWidth="60" prefWidth="150" />
+      <ColumnConstraints hgrow="ALWAYS" percentWidth="40" prefWidth="150" />
```

Comment:
Just curious.. these 2 lines look the same so I wonder what do they do.

Reply:
I am doing percentWidth=60 as the ratio of two columns in one person card will be 6:4

Reply:
Ahh I see thxx